Avoid runtime changes in /var/lib/dbus/machine-id.

adelton opened the pull request:
Untested, should be mostly equivalent to https://github.com/freeipa/freeipa-container/pull/161.

This should make `docker diff` pass, at least on Fedora 25 hosts.

stlaz commented:
I'll try to test this ASAP.

stlaz commented:
The change in the rawhide Dockerfile makes the image unbuildable as it seems that `/var/lib/dbus/machine-id` already exists in the base image, which I guess is actually kind of weird.
The change in the Fedora 26 Dockerfile works fine, though.

adelton commented:
Ah, `fedora:rawhide` does not have it but `dnf upgrade -y` that we run in `Dockerfile.fedora-rawhide` brings the symlink. Sorry about that.

Fixed the pull request to only bring the change to `Dockerfile.fedora-26`, rebased on master -> 7e52f25d311ad86c106f35264b057d503e8d20e5.

stlaz commented:
Thanks, ACK!
